frontend/src/concepts/hardwareProfiles/HardwareProfileTableColumn.tsx

```diff
@@ -0,0 +1,90 @@
+import { HelperText, HelperTextItem, Spinner, Flex, FlexItem } from '@patternfly/react-core';
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { ModelResourceType } from '@odh-dashboard/model-serving/extension-points';
+import { SupportedArea, useIsAreaAvailable } from '#~/concepts/areas';
+import { NotebookKind } from '#~/k8sTypes';
+import { ContainerResources } from '#~/types';
+import ScopedLabel from '#~/components/ScopedLabel';
+import { ScopedType } from '#~/pages/modelServing/screens/const';
+import { getHardwareProfileDisplayName } from '#~/pages/hardwareProfiles/utils';
+import { resourceTypeOf } from '#~/concepts/hardwareProfiles/utils';
+import { useHardwareProfileBindingState } from './useHardwareProfileBindingState';
+import HardwareProfileDetailsPopover from './HardwareProfileDetailsPopover';
+import HardwareProfileBindingStateLabel from './HardwareProfileBindingStateLabel';
+import { HardwareProfileBindingState } from './const';
+
+type HardwareProfileTableColumnProps = {
+  namespace: string;
+  resource: NotebookKind | ModelResourceType;
+  containerResources: ContainerResources | undefined;
+  isActive?: boolean;
+};
+
+const HardwareProfileTableColumn: React.FC<HardwareProfileTableColumnProps> = ({
+  namespace,
+  resource,
+  containerResources,
+  isActive = false,
+}) => {
+  const isProjectScoped = useIsAreaAvailable(SupportedArea.DS_PROJECT_SCOPED).status;
+  const [bindingStateInfo, bindingStateLoaded, loadError] =
+    useHardwareProfileBindingState(resource);
+
+  const hardwareProfile = bindingStateInfo?.profile;
+
+  if (loadError && bindingStateInfo?.state !== HardwareProfileBindingState.DELETED) {
+    return (
+      <HelperText>
+        <HelperTextItem variant="error">Custom</HelperTextItem>
+      </HelperText>
+    );
+  }
+  if (!bindingStateLoaded) {
+    return <Spinner size="md" />;
+  }
+  const displayName = hardwareProfile ? getHardwareProfileDisplayName(hardwareProfile) : 'Custom';
+  const resourceType = resourceTypeOf(resource);
+  return (
+    <>
+      <Flex
+        spaceItems={{ default: 'spaceItemsSm' }}
+        alignItems={{ default: 'alignItemsCenter' }}
+        data-testid="hardware-profile-table-column"
+      >
+        {bindingStateInfo?.state !== HardwareProfileBindingState.DELETED && (
```

Comment:
so if it is deleted we can still show the HardwareProfileDetailsPopover

it would really just use the container resources but this would give the user visibility that we have here
<img width="385" height="292" alt="Screenshot 2025-09-30 at 4 25 34 PM" src="https://github.com/user-attachments/assets/8eef8939-87c9-4ba8-83a6-634f498be405" />


now u have on hover for it to show a general deleted message, but we could also have the resource exisiting settings as well. 

this was not in the mocks so i wont block the pr on this. but ill tag @vconzola and maybe we can circle back in a later release on this